### DISCUSSION TITLE
Suggestion for intel feed

### DISCUSSION THREAD

**Discussion by tsmullins** [Security-Onion-Solutions/securityonion#12153] [General]
What intel feeds are people using?  Any suggestions for a particular feed.

Thanks
Shane

**Comment by InfosecGoon** [comment]
One of the best sources of intel is the ISAC for your industry - they've usually got some high-fidelity stuff for threats specific to you.

Is this for an enterprise deployment or something at home?

**Comment by KimkimCyber** [comment]
ETPro and MISP
